src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -397,85 +347,7 @@ static int gethostbyaddr_r(const uint8_t* addr, const socklen_t len, int type, s
 #define HAVE_GETHOSTBYADDR_R 1
```

Comment:
previous line (346 `#define HAVE_GETHOSTBYNAME_R 1`) can be removed. Mac build is failing because it's unused: https://ci3.dot.net/job/dotnet_corefx/job/master/job/osx-TGroup_netcoreapp+CGroup_Debug+AGroup_x64+TestOuter_false_prtest/8291/console

Reply:
Thank's i'm OOF tomorrow i'll do it.